apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -72,6 +72,16 @@ const envVarSchema = z.object({
   CB_VERIFICATION_ATTESTER: z
     .string()
     .default('0x357458739F90461b99789350868CD7CF330Dd7EE'),
+  FAUCET_V1_CONTRACT_ADDRESS: z.string(),
+  FAUCET_V1_AUTH_ADMIN_WALLET_PRIVATE_KEY: z
+    .string()
+    .describe('private key for faucet v1 auth admin'),
+  JSON_RPC_URLS_L1_SEPOLIA: z
+    .string()
+    .array()
+    .refine((arr) => arr.length > 0, {
+      message: 'must pass in one l1 sepolia rpc url as env var',
+    }),
```

Comment:
to take after the branch we used to test the new faucet, you will also want to add the on chain and off chain module addresses to the env vars: https://github.com/ethereum-optimism/gateway/compare/develop...test_new_faucet#diff-cbde86bcf12cb3137d6c6fd0e10bdc13deb4762d41bb87b47548d60f605c3722R147-R148